docs/website/docs/general-usage/credentials/configuration.md

```diff
@@ -366,12 +379,9 @@ private_key = <private_key from services.json>
 project_id = <project_id from services json>
 ```
 
-Now when `dlt` looks for destination credentials, it will encounter the `destination` section and
-stop there. When looking for `sources` credentials it will get directly into `credentials` key
-(corresponding to function argument).
+Now when `dlt` looks for destination credentials, it will start with **destination.bigquery.credentials**, eliminate **bigquery** and stop at **destination.credentials**
 
-> We could also rename the argument in the source function! But then we are **forcing** the user to
-> have two copies of credentials.
+When looking for `sources` credentials it will start with **sources.google_sheets.google_sheets.credentials**, eliminate **google_sheets** twice and stop at **sources.credentials** (we assume that `google_sheets` source was defined in `google_sheets` python module)
```

Comment:
```suggestion
When looking for `sources` credentials it will start with `sources.google_sheets.google_sheets.credentials`, eliminate `google_sheets` twice and stop at `sources.credentials` (we assume that `google_sheets` source was defined in `google_sheets` python module)
```